src/model/enums/misc.rs

```diff
@@ -91,3 +93,20 @@ pub enum Modality {
     Major = 1,
     NoResult = -1,
 }
+/// Limit the response to a particular market
+///
+/// FromToken is the same thing as setting the market parameter to the user's country.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+pub enum Market {
+    Country(Country),
+    FromToken,
+}
+
+impl Into<String> for Market {
```

Comment:
Should this be moved somewhere else in order to keep this file to only declarations of enums?

Reply:
Nah I would say it's ok

Reply:
As as side note: can you add a new line between the previous enum and this one? Thanks